src/ert/gui/model/node.py

```diff
@@ -73,8 +71,8 @@ def __repr__(self) -> str:
 class RealNodeData:
     status: str | None = None
     active: bool | None = False
-    fm_step_status_color_by_id: dict[str, QColor] = field(default_factory=dict)
-    real_status_color: QColor | None = None
+    fm_step_status_color_by_id: dict[str, str] = field(default_factory=dict)
```

Comment:
"color" should maybe not be part of the variable name since there is only descriptive strings for states in the dict?

Reply:
Agreed. I've renamed variables that do not contain color anymore.